metatron-app/metatron-discovery
# #1907 Chart selection filter error

### Description

1. When the data of the date type is individually selected, the filter condition refers to the data in the range.
2. Color changed when Gauge chart selection filter.
3. When there is only one chart on the dashboard, when you apply the selection filter, it is invisible and appears when you click on dashboard

**Related Issue** : #1907 


### How Has This Been Tested?

1. When data of the date type is individually selected, check that the filter conditions are set individually.
2. Make sure that the color representation for the chart selection is working properly.
3. Make sure the selection filter is displayed when there is only one chart in the dashboard.

#### Need additional checks?
You need to check the selection filter behavior for other charts.

### Types of changes



### Checklist:


### Additional Context
